Frontend/CO.CDP.OrganisationApp/Pages/Users/UserCheckAnswers.cshtml.cs

```diff
@@ -41,20 +40,26 @@ public async Task<IActionResult> OnPost()
             return RedirectToPage("AddUser", new { Id });
         }
 
-        var personInviteCommand = new InvitePersonToOrganisation(
-            PersonInviteStateData.Email,
-            PersonInviteStateData.FirstName,
-            PersonInviteStateData.LastName,
-            PersonInviteStateData.Scopes
-        );
-
-        await organisationClient.CreatePersonInviteAsync(Id, personInviteCommand);
+        try
+        {
+            var personInviteCommand = new InvitePersonToOrganisation(
+                PersonInviteStateData.Email,
+                PersonInviteStateData.FirstName,
+                PersonInviteStateData.LastName,
+                PersonInviteStateData.Scopes
+            );
 
-        session.Remove(PersonInviteState.TempDataKey);
+            await organisationClient.CreatePersonInviteAsync(Id, personInviteCommand);
 
-        tempDataService.Put(FlashMessageTypes.Success, $"You've sent an email invite to {PersonInviteStateData.FirstName} {PersonInviteStateData.LastName}");
```

Comment:
`tempDataService` is no longer used and can be removed from the constructor.

Reply:
Done